src/App.tsx

```diff
@@ -14,7 +29,7 @@ export const App = () => {
       </div>
 
       <div className="sidebar">
-        <FindMovie />
+        <FindMovie onAdd={addMovie} />
```

Comment:
I would only pass addMovie and here and change the name of the prop to setMovies signifying function changing state.